app/assets/javascripts/application/tags.js

```diff
@@ -41,4 +45,26 @@ $(function() {
       }
     }
   });
+
+  $('#tags-js-members').select2({
```

Comment:
I'd delete this new initializer to make it DRYer. The only difference is the param you need to send to the backend, which can be pulled from a data-attribute, for example.